SingularityUI/app/actions/api/base.es6

```diff
@@ -1,5 +1,22 @@
 import fetch from 'isomorphic-fetch';
 
+export default function buildPostJsonApiAction(actionName, opts={}) {
+  let jsonBoilerplate = {
+    method: 'POST',
+    headers: {'Content-Type': 'application/json', 'Accept': 'application/json'}
+  }
```

Comment:
True, but not if we want to be able to use methods other than POST.
`headers` can still be a `const` though